dockers/skylab/build-indices/docker_versions.tsv

```diff
@@ -0,0 +1,2 @@
+us.gcr.io/broad-gotc-prod/build-indices:1.0.0-2.7.10a-1663343355
+us.gcr.io/broad-gotc-prod/build-indices:1.0.0-2.7.10a-1663605340
```

Comment:
For uniformity with the other images add `DOCKER_VERSION` to the topmost line and remove older versions not in use

Reply:
Done.